vcl_templates/www.vcl.erb

```diff
@@ -220,6 +220,10 @@ sub vcl_recv {
     set req.http.GOVUK-ABTest-EducationNavigation = "A";
   } else if (req.url ~ "[\?\&]ABTest-EducationNavigation=B(&|$)") {
     set req.http.GOVUK-ABTest-EducationNavigation = "B";
+  } else if (req.url ~ "^\/education(\/)?.*$") {
```

Comment:
I think we need an equivalent check in the delivery step to make sure the B cookie gets set. Otherwise this will show the user the B version on this request but will preserve their original cookie.